### DISCUSSION TITLE
Deprecation of session response in extend API

### DISCUSSION THREAD

**Discussion by trautonen** [ory/kratos#4132] [Q&A] [answered]
The v1.3.0 [changelog](https://github.com/ory/kratos/releases/tag/v1.3.0) states:

> Going forward, the /admin/session/.../extend endpoint will return 204 no content for new Ory Network projects. We will deprecate returning 200 + session body in the future.

We are currently using the extend API automatically for active users and the backend updates the new extended session details to the response so that the client knows also when it expires. How this should be done if the extend API does not return the session details anymore?

**Comment by aeneasr** [comment, marked as answer]
You can fetch the session after expiry, unfortunately there is no other way to do it.
